e2e/npm_translate_lock/repositories_checked.bzl

```diff
@@ -1,9 +1,9 @@
-"@generated by @aspect_rules_js//npm/private:npm_translate_lock.bzl from //:pnpm-lock.yaml"
+"@generated by npm_translate_lock(name = 'npm')"
 
 load("@aspect_rules_js//npm:npm_import.bzl", "npm_import")
 
 def npm_repositories():
-    "Generated npm_import repository rules corresponding to npm packages in //:pnpm-lock.yaml"
```

Comment:
Isn't this still true?

Reply:
Yea. The comments were in flux here but I think it can go back to the pnpm-lock.yaml label again now that the code always depends on one being in the source tree even in the `pnpm import` case